README.md

```diff
@@ -51,36 +49,9 @@ var client = new WorkOSClient(
 WorkOS.WorkOSClient = client;
 ```
 
-## Example Usage
-
-The example below demonstrates exchanging an authorization code for a WorkOS Profile.
-
-```c#
-var ssoService = new SSOService();
-var options = new GetProfileOptions
-{
-    ClientId = "project_123",
-    Code = "code_123",
-};
-var response = ssoService.GetProfile(options);
-```
-
-When applicable, calls can be made asynchronously and return a `Task`.
-
-```c#
-var response = await ssoService.GetProfileAsync(options);
-```
+## More Information
 
-## Development and Testing
```

Comment:
We should either leave the contribution docs in the README, or move them to a separate `CONTRIBUTING.md`.

This will help developers who want to contribute to the SDK get started.